models/attribute-definition/package.json

```diff
@@ -8,22 +8,31 @@
     "url": "https://github.com/commercetools/test-data.git",
     "directory": "models/attribute-definition"
   },
-  "keywords": ["javascript", "typescript", "test-data"],
+  "keywords": [
+    "javascript",
+    "typescript",
+    "test-data"
+  ],
   "license": "MIT",
   "publishConfig": {
     "access": "public"
   },
   "main": "dist/commercetools-test-data-attribute-definition.cjs.js",
   "module": "dist/commercetools-test-data-attribute-definition.esm.js",
-  "files": ["dist", "package.json", "LICENSE", "README.md"],
+  "files": [
+    "dist",
+    "package.json",
+    "LICENSE",
+    "README.md"
+  ],
   "dependencies": {
     "@babel/runtime": "^7.17.9",
     "@babel/runtime-corejs3": "^7.17.9",
     "@commercetools-test-data/attribute-type": "5.8.2",
     "@commercetools-test-data/commons": "5.8.2",
     "@commercetools-test-data/core": "5.8.2",
     "@commercetools-test-data/utils": "5.8.2",
-    "@commercetools/platform-sdk": "^4.0.0",
+    "@commercetools/platform-sdk": "^4.11.0",
```

Comment:
the caret here [should mean](https://classic.yarnpkg.com/lang/en/docs/dependency-versions/#toc-caret-ranges) that when you run yarn it will install the latest minor version of `4.x.x`, was that not happening here?